### ISSUE TITLE
App doesnt keep you logged in at all

### ISSUE THREAD

**Issue by rmtodd618** [NewEconomyMovement/NEMAndroidApp#158]
Dont know if that is a feature or if it hasnt been resolved yet but I think if a phone has a security password to log into the phone the app should not require your password EVERYtime you want to login.

**Comment by rmtodd618** [comment]
version 0.1.36.b2565

**Comment by rmtodd618** [comment]
version 0.1.36.b2565
LG G3
android 5.0.1
kernel version 3.4.0

**Comment by jabo38** [comment]
I agree and this has been discussed before a few times but there is no consensus yet on it.  Ideally there needs to be a setting in Configurations for this option I guess.

**Comment by kodty** [comment]
im finding it very annoying to have to log in over and over. it will just cause people to use very basic passwords. a setting to turn on or off repeat sign in would be a good update.
